package FogGateConfig;

message FogGate
{
	required uint32 gateid		= 1;
	required uint32 terrain		= 2;
	repeated uint32 npcs		= 3;
}

message FogGates
{
	repeated FogGate foggates	= 1;
}